### PR TITLE
Set up a k8s cluster as a turnkey dev environment

### PR DESCRIPTION
**Pull request by tylershelton**
This PR does a few things, high level --

1. It sets up the necessary files for building docker images of Kubernautics,
2. It creates a k8s cluster config suitable for use as a dev environment,
3. Adds some scripting to make that dev environment easy to setup and tear down. This script is run by invoking `npm run cluster (up|down)`, and
4. The process to setup this dev cluster and get the project running in development mode is documented in the project's README.

### CODE REVIEW

File: README.md
Comment (edli-dot) [inline README.md:1]: followed instructions. about 12 min set up time. it does check for minikube present, if not it starts minikube

File: scripts/dev_cluster.sh
Comment (edli-dot) [inline scripts/dev_cluster.sh:1]: dev_cluster up and down logic work. dev_cluster will require an additional argument. all works

File: README.md
Comment (edli-dot) [reply README.md:1]: though at the very bottom of the dev_cluster.sh, I would specify how to exit out of the devspace: "type exit in devspace terminal to exit out of devspace". this wasn't explicit.

**Review by edli-dot — CHANGES_REQUESTED** [review]
edit read me to include exit statement to get out of devspace

File: README.md
Comment (tylershelton) [reply README.md:1]: Great, I made changes to `README.md` and `devspace_start.sh` to make that clearer.

**Review by edli-dot — APPROVED** [review]
Looks good now

**Pull request merged**